mongodb-js/compass
# COMPASS-1160 feature tour 1.7

To verify: 

1. start Compass, open dev console, enter: `app.preferences.save({lastKnownVersion: "1.6.2"})`
2. quit Compass, restart

The feature tour should now show the 2 new features as if someone upgraded from 1.6 to 1.7
